public/js/script.js

```diff
@@ -0,0 +1,19 @@
+// JavaScript for disabling form submissions if there are invalid fields
```

Comment:
Instead of using the comment you could be communicate that by naming the function and the file. Currently the file is called `script.js` which is very generic.

Suggestion: you could name the function `validateForm` and the file `validate-form.js`